src/UI/Component/Panel/Sub.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/* Copyright (c) 2016 Timon Amstutz <timon.amstutz@ilub.unibe.ch> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Panel;
+
+/**
+ * This describes a Sub Panel.
+ */
+interface Sub extends Panel {
```

Comment:
I'm not sure, i understand what you are asking. You could introduce `interface PanelWithContent` (proper naming pending) that has the `*Content` methods. This would deduplicate the `*Content` methods in Sub and Panel.